client/app/scripts/superdesk-desks/views/aggregate-widget.html

```diff
@@ -2,12 +2,25 @@
     <div class="search-container">
         <div class="search-box">
             <input type="text" placeholder="{{ 'Search'|translate }}" ng-model="query">

```

Comment:
Fixed strings like this one should use the one-time binding syntax (::) to reduce the number of $watches.